### PR TITLE
CI-171: Sometimes the Emotion word bubbles are not located in appropriate emotion categories

### PR DESCRIPTION
**Pull request by VitaliiShevchenko**
[CI-171](https://cloverpop-internship.atlassian.net/browse/CI-171)

[CI-171]: https://cloverpop-internship.atlassian.net/browse/CI-171?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -103,7 +103,6 @@
     t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
   end
 
-  add_foreign_key "responses", "emotions"
```
Comment (SerhiiBorozenets) [inline db/schema.rb:106]: @VitaliiShevchenko sorry, but I don't understand why you remove this `foreign_key`? I don't see any code that could cause this.

**Review by SerhiiBorozenets — CHANGES_REQUESTED** [review]

File: db/schema.rb (hunk 1 above)
Comment (VitaliiShevchenko) [reply db/schema.rb:106]: @SerhiiBorozenets  This is an auto-generated comment dependent on migrations that are not related to this PR

Comment (SerhiiBorozenets) [reply db/schema.rb:106]: @VitaliiShevchenko you should run migration locally in this PR and update PR

Comment (VitaliiShevchenko) [reply db/schema.rb:106]: image

Comment (SerhiiBorozenets) [reply db/schema.rb:106]: @VitaliiShevchenko I pulled this PR, run migration and I have changes. Try drop your db and run migration again.

**Review by SerhiiBorozenets — APPROVED** [review]

File: app/javascript/components/Pages/ListEmotions.js [hunk 2]
```diff
@@ -80,6 +80,7 @@ function ListEmotions({ data,  setData , saveDataToDb, steps, service}) {
           <div className="mx-auto my-0 question">Which word best describes how you felt at work this week?</div>
             <div className='d-flex mx-auto emotions'>
                 {emotions.map((emotion, index) =>
+                  <div style={{width: 180}}>
```
Comment (SokalMet) [inline app/javascript/components/Pages/ListEmotions.js:83]: You should avoid using such 'style' implementation. Remove it pls and use class name with scss style

**Review by SokalMet — APPROVED** [review]

**Pull request merged**